vpr/src/route/rr_graph.h

```diff
@@ -55,6 +55,44 @@ void load_rr_switch_from_arch_switch(int arch_switch_idx,
                                      const float R_minW_nmos,
                                      const float R_minW_pmos);
 
+// Class for building a group of connected edges.
```

Comment:
With this refactoring, is there any reason for this class to live in rr_graph.h?

Reply:
I've moved the class to its own files.